### ISSUE TITLE
absolute path in Julia example

### ISSUE THREAD

**Issue by matbesancon** [ERGO-Code/HiGHS#213]
in `examples/call_highs_from_julia.jl:29`
```julia
Highs_readFromFile(highs, "/home/s1613957/HiGHs/build/bin/avgas.mps")
```

The absolute path is not ideal, and the mps file is not present after building the project

**Comment by feldmeier** [comment]
thanks for pointing this out. I will change it to a relative path pointing to one of the instances in the the /check folder.

**Comment by feldmeier** [comment]
This will be fixed by Pullrequest #214
